WinBuild.cmd

```diff
@@ -0,0 +1,43 @@
+@echo off
+@setlocal EnableDelayedExpansion
+
```

Comment:
Could you either remove this from the PR or move it into e.g. a `contrib/` folder? I have no way of testing it, and I don't want to be responsible for updating three different build systems whenever I want to make a change to one of them. (Same comment mostly applies to the CMakeLists.txt)